test/e2e/user_management_test.go

```diff
@@ -68,6 +69,35 @@ func (s *userManagementTestSuite) TestUserDeactivation() {
 		_, err = s.hostAwait.WaitForMasterUserRecord(mur.Name)
 		require.NoError(s.T(), err)
 	})
+
+	s.T().Run("check that auto deactivation deactivates a user", func(t *testing.T) {
+		tierDeactivationPeriod := 30
+		ctx, hostAwait, _ := WaitForDeployments(t, &v1alpha1.NSTemplateTier{})
+		defer ctx.Cleanup()
+
+		// Let's create a tier with deactivation enabled
+		deactivationTier := CreateNSTemplateTier(t, ctx, hostAwait, "deactivation-tier", DeactivationTimeoutDays(tierDeactivationPeriod))
+
+		mur := MoveUserToTier(t, hostAwait, userSignup.Spec.Username, *deactivationTier)
+
+		err := s.hostAwait.Client.Get(context.TODO(), types.NamespacedName{
+			Namespace: mur.Namespace,
+			Name:      mur.Name,
+		}, mur)
```

Comment:
Why do you need to get Mur again here? Is something wrong with the mur you got from `MoveUserToTier()`?

Reply:
Oops, that's a mistake. Forgot to remove that after refactoring the code. Fixed in https://github.com/codeready-toolchain/toolchain-e2e/pull/162/commits/284053b155da49bcbd952b32e2ee8f5da4ba211f